activesupport/lib/active_support/core_ext/module/delegation.rb

```diff
@@ -190,6 +196,16 @@ def delegate(*methods, to: nil, prefix: nil, allow_nil: nil, private: nil)
     to = to.to_s
     to = "self.#{to}" if DELEGATION_RESERVED_METHOD_NAMES.include?(to)
 
+    if prefixed
+      methods = methods.map do |method|
+        method.to_s.sub!(/^#{method_prefix}/, '')&.to_sym
```

Comment:
You can use `delete_prefix!` since Ruby 2.5.0.

Reply:
Thanks. Fixed it (82c2e54a9db416c495f6b8c51827649d732d05e8).